lib/craft/machine.ex

```diff
@@ -136,6 +136,11 @@ defmodule Craft.Machine do
     |> GenServer.call(request, timeout)
   end
 
+  @doc "Allows responding to a query from a different process, like GenServer.reply"
+  def reply({pid, query_from}, reply) do
```

Comment:
if the query was eventually consistent, we don't want to send the results back to the machine to wait for a quorum round to flush the result to the client. the process doing the query can send it directly to the client.

adding a flag to the `from` with the consistency level would allow this function to differentiate. like, `{machine_pid, consistency, from}`